build/android/bytecode/java/org/brave/bytecode/BraveFreIntentCreatorClassAdapter.java

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.brave.bytecode;
+
+import org.objectweb.asm.ClassVisitor;
+
+public class BraveFreIntentCreatorClassAdapter extends BraveClassVisitor {
+    static String sFreIntentCreatorClassName =
+            "org/chromium/chrome/browser/firstrun/FreIntentCreator";
+
+    static String sBraveFreIntentCreatorClassName =
+            "org/chromium/chrome/browser/firstrun/BraveFreIntentCreator";
+
+    static String sMethodReadFileAsUtf8 = "createInternal";
+
+    public BraveFreIntentCreatorClassAdapter(ClassVisitor visitor) {
+        super(visitor);
+
+        changeMethodOwner(
```

Comment:
We don't need this. We need to change method owner for static functions only. For members we just override them.